react/utils/pages.ts

```diff
@@ -154,29 +154,36 @@ export function navigate(
   const {
     page,
     params,
-    query,
-    to,
+    query: inputQuery,
+    to: inputTo = '',
```

Comment:
Is this default value really neccessary? Because you're setting `inputTo = ''` but later you verify if the value is falsy

Reply:
Oh, I see, this string is used later to split the values, but is it possible that this variable can receive `null`?

Reply:
Yes, than it will return on the first check